packages/web-component/test/descope-wc.test.ts

```diff
@@ -618,6 +618,37 @@ describe('web-component', () => {
     await waitFor(() => expect(nextMock).toHaveBeenCalled());
   });
 
+  it('When there is a single "sso" button and pressing on enter, it clicks the button', async () => {
+    startMock.mockReturnValueOnce(generateSdkResponse());
+    nextMock.mockReturnValueOnce(generateSdkResponse({ screenId: '1' }));
+
+    pageContent =
```

Comment:
I would add a couple of SSO buttons to see the first one is picked, and validate that if there are multiple buttons (some might be not connected) the the first is always picked.

Reply:
If there are many non should be picked, will add a test for that.

Reply:
added